server/controllers/base.ts

```diff
@@ -4,58 +4,95 @@ abstract class BaseCtrl {
 
   // Get all
   getAll = (req, res) => {
-    this.model.find({}, (err, docs) => {
-      if (err) { return console.error(err); }
-      res.json(docs);
-    });
-  }
+    if (!req.payload.user._id) {
+      res.status(401).json({
+        'message' : 'UnauthorizedError: private'
+      });
```

Comment:
These 4 lines (7-10) are repeated throughout the entire file. There is surely a better way to write this part, in a way to respect the DRY principle.

Reply:
hi did you give a look at my pull request

Reply:
i am trying to say that i have already implemented similar kind of workout